### DISCUSSION TITLE
autoComplete from event handler

### DISCUSSION THREAD

**Discussion by McFrag** [TarekRaafat/autoComplete.js#283] [Q&A]
Hi there.
Is there a way to get the autoComplete in the input event handler?
For example the someFunction in the input.keydown event below.

```javascript
events: 
{
      input:
        {
             keydown: (event) =>
                {
                    var ctl = someFunction(event);
                    if (!ctl.isOpen) doStuff();
                 }
        }
}
```

**Comment by folknor** [comment]
Yes, you just need to bind it yourself to a property on the input.

**Reply by sanjay-tiwari-tfs** [reply]
Hi @folknor , How to do .. can you please add an example.
